apps/prairielearn/src/pages/partials/question.ejs

```diff
@@ -205,3 +213,69 @@
   <% } %> <%# endif showSubmissions %>
 
 </div>
+
+<div id="copyQuestionModal" class="modal fade" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <form method="POST">
+        <div class="modal-header">
+          <h5 class="modal-title">Copy question</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body">
+          <% if (question_copy_targets?.length === 0) { %>
+            <p>
+              You can&apos;t copy this question because you don&apos;t have editor permissions in any courses.
+              <a href="/pl/request_course">Request a course</a> if you don&apos;t have one already.
+              Otherwise, contact the owner of the course you expected to have access to.
+            </p>
```

Comment:
Mariana had the great idea to integrate a CTA if the user isn't the instructor of any courses yet!